manke/htcondor/htcondor-worker-deployment.yaml

```diff
@@ -0,0 +1,52 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: htcondor-worker
+  namespace: htcondor
+  labels:
+    app: htcondor-worker
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      app: htcondor-worker
+  template:
+    metadata:
+      annotations:
+        k8s.v1.cni.cncf.io/networks: kube-system/htcondor
+      labels:
+        app: htcondor-worker
+    spec:
+      containers:
+      - name: htcondor-worker
+        image: docker.io/lsstit/htcondor-worker:23
+        imagePullPolicy: Always
+        env:
+        - name: CONDOR_HOST
+          value: "139.229.152.211"
```

Comment:
There's no PTR record for this address. Could you add one?  Is there an FQDN that could be used instead of an IP?